### DISCUSSION TITLE
Fallen at the first hurdle

### DISCUSSION THREAD

**Discussion by chesterflaps** [johanmeijer/grott#389] [Q&A]
Hi all.

I have no idea how I install this in to HA. I thought it would be a custom repository in HACS but it says it is not compliant.

what am I missing?

**Comment by FreezyExp** [comment]
what is your setup? what steps have you taken?

we are missing a lot of information

**Reply by chesterflaps** [reply]
At the point I posted, I had done nothing because I couldn't work out how to get the code. I found it though.

**Comment by JazekerXX** [comment]
> At the point I posted, I had done nothing because I couldn't work out how to get the code. I found it though.


could you explain how you installed it? i only see how to install this on the docker version of HA, is it possible to install this on the supervised HA version?
